PowerPointLabs/Test/UnitTest/TooltipsLab/CreateTooltipTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using PowerPointLabs.TooltipsLab;
+using PowerPointLabs.Models;
+
+using Test.Util;
+using Microsoft.Office.Interop.PowerPoint;
+
+namespace Test.UnitTest.TooltipsLab
+{
+    [TestClass]
+    public class CreateTooltipTest : BaseUnitTest
+    {
+        private const int CreateTooltipNoneSelectedTestSlideNo = 4;
+        private const int CreateTooltipNoneSelectedExpectedSlideNo = 5;
+        private const int CreateTooltipShapeSelectedTestSlideNo = 7;
+        private const int CreateTooltipShapeSelectedExpectedSlideNo = 8;
+        private const int CreateTooltipExistingTooltipTestSlideNo = 10;
+        private const int CreateTooltipExistingTooltipExpectedSlideNo = 11;
+        private const int CreateTooltipMultipleTooltipsTestSlideNo = 13;
+        private const int CreateTooltipMultipleTooltipsExpectedSlideNo = 14;
+
+        private const string CreateTooltipShapeToSelectName = "SelectMe";
+
+        protected override string GetTestingSlideName()
+        {
+            return "TooltipsLab\\CreateTooltip.pptx";
+        }
+
+        [TestMethod]
+        [TestCategory("UT")]
+        public void CreateTooltip()
+        {
+            TestCreateTooltip_NoneSelected();
+            TestCreateTooltip_ShapeSelected();
+            TestCreateTooltip_ExistingTooltip();
+            TestCreateTooltip_MultipleTooltips();
+        }
+
+        private void TestCreateTooltip_NoneSelected()
+        {
+            PpOperations.SelectSlide(CreateTooltipNoneSelectedTestSlideNo);
+            CreateTooltipAndCompare(CreateTooltipNoneSelectedTestSlideNo, CreateTooltipNoneSelectedExpectedSlideNo);
+        }
+
+        private void TestCreateTooltip_ShapeSelected()
+        {
+            PpOperations.SelectSlide(CreateTooltipShapeSelectedTestSlideNo);
+            PpOperations.SelectShape(CreateTooltipShapeToSelectName);
+            CreateTooltipAndCompare(CreateTooltipShapeSelectedTestSlideNo, CreateTooltipShapeSelectedExpectedSlideNo);
+        }
+       
+        private void TestCreateTooltip_ExistingTooltip()
+        {
+            PpOperations.SelectSlide(CreateTooltipExistingTooltipTestSlideNo);
+            CreateTooltipAndCompare(CreateTooltipExistingTooltipTestSlideNo, CreateTooltipExistingTooltipExpectedSlideNo);
+        }
+
+        private void TestCreateTooltip_MultipleTooltips()
+        {
+            PpOperations.SelectSlide(CreateTooltipMultipleTooltipsTestSlideNo);
+            for (int i = 0; i < 5; i++)
```

Comment:
the `5` could be in a variable like number of tooltips

Reply:
Done!